README.md

```diff
@@ -15,6 +15,8 @@ After `xmtpd` meets specific functional requirements, the plan is for it to beco
 - [Go](https://go.dev/doc/install)
 - [Docker](https://www.docker.com/get-started/)
 
+You must have the _exact_ go version listed in `go.mod` - you can verify this by running `go version`.
```

Comment:
Of all our projects, this is probably the one that needs a devcontainer the most come to think of it.